config/jobs/kubernetes-sigs/image-builder/image-builder-ci.yaml

```diff
@@ -0,0 +1,30 @@
+postsubmits:
+  kubernetes-sigs/image-builder-konfigadm:
+    - name: post-image-builder-konfigadm
+      labels:
+        preset-dind-enabled: "true"
+        preset-image-builder-gcp-creds: "true"
+        preset-image-builder-aws-creds: "true"
+        preset-image-builder-azure-creds: "true"
+        preset-image-builder-vsphere-creds: "true"
+      branches:
+        - ^master$
+      always_run: false
+      run_if_changed: "^images/konfigadm/"
+      decorate: true
+      path_alias: sigs.k8s.io/image-builder
+      max_concurrency: 1
+      spec:
+        containers:
+          - image: google/cloud-sdk:264.0.0
```

Comment:
why use this image and not e.g. kubekins?

Reply:
for this job it wouldn't be doing anything but spinning up a new nested virtualization VM and running the tests from their